typegate/src/runtimes/prisma.ts

```diff
@@ -37,6 +42,59 @@ function isPrismaOperationMat(mat: Materializer): mat is PrismaOperationMat {
   return mat.name === "prisma_operation";
 }
 
+export interface PrismaRuntimeDS extends Omit<TGRuntime, "data"> {
+  data: PrismaRuntimeData;
+}
+
+registerHook("onPush", async (typegraph) => {
```

Comment:
I believe you may need to introduce "push" parameter (from the CLI) as one may want to push without apply the migration (and handle migration separately)

Reply:
:+1:
I will add `--no-migrations` which will skip the addition of the migrations files in the serialized typegraph...